presto-docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -159,6 +159,19 @@ For example, the mapping list might have URL prefix ``s3://abc/xyz/`` followed b
 than for other paths within the bucket. You can set default configuration by not
 including any match criteria for the last entry in the list.
 
+In addition to the rules above, the default mapping can contain the
+``userClusterDefault`` property set to ``true``. This causes the the default
```

Comment:
It can be set to false as well. Did you mean 'property when set to'?

Reply:
I've extended the description.